bots/tests-scan

```diff
@@ -138,23 +131,26 @@ def main():
 
     api = github.GitHub(repo=opts.repo)
 
+    # This options flow around the code, for example repo from opts is needed in
+    # `tests_invoke`, `tests_human`, `queue_test` etc.
+    opts.repo = api.repo
```

Comment:
I just did it this way and commented why it happens, we can clean it up later on.